apps/vr-tests-react-components/package.json

```diff
@@ -5,15 +5,14 @@
   "description": "Visual regression tests for @fluentui/react-components",
   "scripts": {
     "build": "build-storybook -o dist/storybook",
-    "test:component": "storywright  --browsers chromium --url dist/storybook --destpath dist/screenshots --waitTimeScreenshot 500 --concurrency 4 --headless true",
     "convert": "bash convertStoriesToStoryWright.sh",
     "clean": "just-scripts clean",
     "format": "prettier . -w --ignore-path ../../.prettierignore",
     "lint": "just-scripts lint",
-    "screener": "just-scripts screener",
     "screener:build": "yarn build",
```

Comment:
`screener:build` will be renamed in a separate PR along with other miscellaneous changes